src/gt4py/backend/templates/dawn_computation.src.in

```diff
@@ -0,0 +1,141 @@
+{#
```

Comment:
There is a lot of redundancy between  `dawn_computation.src.in` and `computation.src.in` maybe some of it could be reduced in the future by changing the latter?

Reply:
Yes, in general there is a lot of redundancy in all the templates for backends using c++ code generation. For the future the main idea is to define a standard API for the generated code and probably then also change the overall design of the backend package. Until then, it's tricky to merge similar-but-not-equal templates because this usually implies more convoluted logic and redundancy in the python side. An intermediate approach would be to use Jinja template inheritance to create a clear structure of templates (also for the GPU backends). I vote for leave it as it is right now, and start clearing the path towards the future standard API using template inheritance in the short term.